### ISSUE TITLE
`any` being returned

### ISSUE THREAD

**Issue by ImLunaHey** [sstur/nbit#1]
This should really be typed as `unknown` instead of `any` as `any` disables type checking.

```ts
import { createApplication } from '@nbit/bun';

const { defineRoutes, attachRoutes } = createApplication();

const routes = defineRoutes(app => [
    app.put('/', request => {
        const body = request.body;
        return body; // This is any
    }),
]);
```

**Comment by ImLunaHey** [comment]
Actually from checking it seems `body` is a readable stream? 🤔

**Comment by sstur** [comment]
You're right, I believe request.body is a readable stream and the typings might be wrong for this, I'll check.

Note: if you're looking for the body to be parsed as JSON, you should use:
```ts
  const data = await request.json();
```

**Comment by sstur** [comment]
Done. This is now fixed in v0.8.0
